core/orch_test.go

```diff
@@ -1181,16 +1182,13 @@ func TestTicketParams(t *testing.T) {
 		},
 	}
 
-	multiplier := big.NewRat(1, 1)
-
 	sender := pm.RandAddress()
-	recipient.On("TxCostMultiplier", mock.Anything).Return(multiplier, nil).Once()
 	recipient.On("TicketParams", mock.Anything, mock.Anything).Return(expectedParams, nil).Once()
 	orch := NewOrchestrator(n, nil)
 
 	assert := assert.New(t)
 
-	actualParams, err := orch.TicketParams(sender)
+	actualParams, err := orch.TicketParams(sender, priceInfo)
```

Comment:
Can we add a check that the underlying `recipient.TicketParams()` method is invoked with the `big.Rat` representation of `priceInfo` to be certain that the value of `priceInfo` is used in `recipient.TicketParams()`? Should be able to use `mock.AssertCalled()` for this.

Reply:
fixed in 5fc1c93e